packages/ui/src/components/theme-toggle.tsx

```
@@ -0,0 +1,86 @@
+
+  if (mode === "light-dark") {
+    const value = mounted ? resolvedTheme : null;
+
+    return (
+      <button
+        className={container}
+        aria-label={`Toggle Theme`}
+        onClick={() => setTheme(value === "light" ? "dark" : "light")}
+        data-theme-toggle=""
+      >
+        {full.map(([key, Icon]) => {
```

Biome (2.4.4)

[error] This callback passed to map() iterable method should always return a value.

(lint/suspicious/useIterableCallbackReturn)